index.js

```diff
@@ -66,7 +66,14 @@ const lintText = async (string, options) => {
 	}
 
 	const report = await eslint.lintText(string, {filePath, warnIgnored});
-	return processReport(report, {isQuiet});
+
+	let rulesMetaForResults;
+
+	if (options && options.rulesMetaForResults === true) {
+		rulesMetaForResults = eslint.getRulesMetaForResults(report);
```

Comment:
Does this call have any noticeable overhead? If not, we could maybe just do it by default without an option.

Reply:
Then I suggest we just do it by default without an option.